src/containers/CreateStream/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import CCard from '../../components/CCard';
+
+const CreateStream = () => {
+  return (
+    <div className="w-[800px]">
+      <CCard
+        className="flex justify-between w-full h-full px-8 py-5"
```

Comment:
CreateStreamCard should use `CPageCard` not `CCard`. Add `title` and `children` properly.